### PR TITLE
Add factorial using PHP

### PR DESCRIPTION
**Pull request by DailyMatters**
Add factorial using PHP as in #9

### CODE REVIEW

File: PHP/factorial.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
```
Comment (utkarsh-shekhar) [inline PHP/factorial.php]: Needs ?> tag as well at the end of the file

Comment (DailyMatters) [reply PHP/factorial.php]: @utkarsh-shekhar I can add it no problem, but the ?> tag is optional.

http://php.net/basic-syntax.instruction-separation

Comment (utkarsh-shekhar) [reply PHP/factorial.php]: Apologies. I just found out that the end tag is optional. Approved and merging this PR

**Comment by utkarsh-shekhar** [comment]
Thanks for contributing and helping me learn new stuff about php! :)

**Comment by DailyMatters** [comment]
Thanks for the opportunity and for creating this repository. It is great for practicing not only git, but also some basic coding. I will continue to contribute!

**Pull request merged**